package.json

```diff
@@ -34,6 +32,8 @@
   "author": "Emmanuel Geoffray",
   "license": "ISC",
   "dependencies": {
+    "font-awesome": "^4.7.0",
+    "signals": "^1.0.0",
```

Comment:
It probably works without. I just mindlessly added it because it was with all the other imported scripts.